### PR TITLE
Cjson writer test suite

### PR DESCRIPTION
**Pull request by Schamnad**
Test cases for the cjson writer implemented during GSOC-2016. 2 test cases:

1. test initialization of the cjson writer
2. test whether the cjson is being dumped in the right manner

### CODE REVIEW

File: test/io/testcjsonwriter.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        self.assertEqual(cjson.ccdata, data)
+
+    def test_cjson_generation(self):
+        """Does the CJSON format get dumped properly"""
+        fpath = os.path.join(__datadir__, "data/ADF/basicADF2007.01/dvb_gopt.adfout")
+        data = cclib.io.ccopen(fpath).parse()
+
+        cjson = cclib.io.cjsonwriter.CJSON(data).generate_repr()
+
+        # if the cjson is generated properly, the data available in the cjson and ccdata
```
Comment (langner) [inline test/io/testcjsonwriter.py]: Please use proper sentences, here and elsewhere.

File: test/io/testcjsonwriter.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        """Does the class initialize correctly?"""
+        fpath = os.path.join(__datadir__, "data/ADF/basicADF2007.01/dvb_gopt.adfout")
+        data = cclib.io.ccopen(fpath).parse()
+        cjson = cclib.io.cjsonwriter.CJSON(data)
+
+        # The object should keep the ccData instance passed to its constructor.
+        self.assertEqual(cjson.ccdata, data)
+
+    def test_cjson_generation(self):
+        """Does the CJSON format get dumped properly"""
```
Comment (langner) [inline test/io/testcjsonwriter.py]: End docstring with a period.

**Comment by langner** [comment]
Would be good to get this in before #332 - please resolve conflicts.

**Comment by Schamnad** [comment]
@langner : have resolved the conflicts, you can merge this before #332 :)

**Pull request merged**